docs/articles/documentation/reference/test-api/testcontroller/getcurrentwindow.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: docs
+title: t.getCurrentWindow Method
+permalink: /documentation/reference/test-api/testcontroller/getcurrentwindow.html
+---
+
+# t.getCurrentWindow method
+
+Retrieves a `window identifier` object that corresponds to the currently open window.
```

Comment:
We usually backtick code entities like variable names, so there's no need for backticks here.